monix-execution/shared/src/main/scala/monix/execution/FutureUtils.scala

```diff
@@ -161,6 +172,10 @@ object FutureUtils {
     p.future
   }
 
+  /** Creates a future that never completes. */
+  def never[A]: Future[A] =
+    Promise[A].future
```

Comment:
@ktoso sorry for the noise, just wanted to mention that your input is greatly appreciated. Thanks.